App/Containers/RootContainer.js

```diff
@@ -7,13 +7,35 @@ import NotificationActions from '../Redux/NotificationRedux'
 import ReduxPersist from '../Config/ReduxPersist'
 import NotificationsBar from '../Components/NotificationsBar'
 import styles from './Styles/RootContainerStyles'
+import Sound from 'react-native-sound'
 
 class RootContainer extends Component {
   componentDidMount () {
     // if redux persist is not active fire startup action
     if (!ReduxPersist.active) {
       this.props.startup()
     }
+    this.coffee = new Sound('coffee.mp3', Sound.MAIN_BUNDLE, (error) => {
```

Comment:
Does this need to live in the RootContainer or could it be moved to the LocationScreen?